deploy/keycloak-auth/scripts/roles-migration.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
It is not migration, so i would rename to roles-configuration.sh

Reply:
We want to test that one role in already included and if so just write ...

Found role `rm-view-all-records`, thus it seems that record manager roles and rolegroups are configured, ignoring this scrit.